File: diploma-service/src/main/java/org/arhor/diploma/service/config/BeansConfig.java
package org.arhor.diploma.service.config;

import org.arhor.commons.util.Converter;
import org.modelmapper.ModelMapper;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class BeansConfig {

  @Bean
  public ModelMapper modelMapper() {
    return new ModelMapper();
  }

  @Bean
  public Converter modelMapperConverter(final ModelMapper mapper) {
    return mapper::map;
  }
}

File: diploma-server/module-dnd/src/main/kotlin/org/arhor/diploma/dnd/BeansConfig.kt
package org.arhor.diploma.dnd

import org.arhor.diploma.commons.file.ChecksumCalc
import org.arhor.diploma.commons.file.ChecksumCalcMD5
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration(proxyBeanMethods = false)
class BeansConfig {

    @Bean
    fun checksumCalc(): ChecksumCalc {
        return ChecksumCalcMD5
    }
}